src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializerFactory.java

```diff
@@ -40,13 +40,39 @@
     private final static Class<?>[] INIT_CAUSE_PARAMS = new Class<?>[] { Throwable.class };
 
     private final static Class<?>[] NO_VIEWS = new Class<?>[0];
-    
+
+
+    /**
+     * Set of well-known "nasty classes", deserialization of which is considered dangerous
+     * and should (and is) prevented by default.
+     */
+    private final static Set<String> DEFAULT_NO_DESER_CLASS_NAMES;
```

Comment:
Made this private just in case someone started using them in 2.6.7.1 only to find they are not present in some versions of 2.7 and 2.8.